kiali-server/templates/role-controlplane.yaml

```diff
@@ -14,4 +14,17 @@ rules:
   verbs:
   - list
 {{- end }}
+{{- if .Values.kiali_feature_flags.certificates_information_indicators.enabled }}
+- apiGroups: [""]
+  resourceNames:
+  {{- range .Values.kiali_feature_flags.certificates_information_indicators.secrets }}
+  - {{ . }}
+  {{- end }}
+  resources:
+  - secrets
+  verbs:
+  - get
+  - list
+  - watch
 ...
+{{- end }}
```

Comment:
This is in the wrong spot. If the feature flags.enabled is false, then the end of file "..." marker isn't included. We want that included all the time, whether the enabled flag is true or not.

So make the final two lines of this file:

```
{{- end }}
...
```

Reply:
Right, thanks, fixed.